migrations/20181202230331-create-profile.js

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
nice work gbols. however, your this migration has no reference to the user model.